lib/Conf.ml

```diff
@@ -935,6 +944,16 @@ module Formatting = struct
       (fun conf x -> {conf with nested_match= x})
       (fun conf -> conf.nested_match)
 
+  let ocaml_version =
+    let docv = "V" in
+    let doc = "Version of OCaml syntax of the output." in
+    let default = Ocaml_version.sys_version in
+    let default_doc = "the currently installed OCaml version" in
```

Comment:
@gpetiot I think that this is not as clear as it could be. Wouldn't "the version of OCaml used to build OCamlFormat" be better?

Reply:
Indeed, done in #1765